ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/ApiTestClientBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using RestSharp;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.TestClients
+{
+    public class ApiTestClientBase : IApiClient
+    {
+        protected RestClient _client;
+
+        public RestClient Client { get => _client; }
```

Comment:
No Need for a getter here, you can just change the methods to do the entire request..and return the response or status code

Reply:
done